src/strategies/meebitsdao/index.ts

```diff
@@ -0,0 +1,62 @@
+import { multicall } from '../../utils';
+import fetch from 'cross-fetch';
+
+export const author = 'peters-josh';
+export const version = '0.1.0';
+
+const abi = [
+  'function ownerOf(uint256 tokenId) public view returns (address owner)',
+  'function tokenURI(uint256 tokenId) public view returns (string uri)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const uriCalls: any[] = [];
+  for (let i = options.startingTokenId; i <= options.endingTokenId; i++) {
+    uriCalls.push([options.address, 'tokenURI', [i]]);
+  }
+
+  const ownerCalls: any[] = [];
+  for (let i = options.startingTokenId; i <= options.endingTokenId; i++) {
+    ownerCalls.push([options.address, 'ownerOf', [i]]);
+  }
+
+  const ownerResponse = await multicall(network, provider, abi, ownerCalls, {
+    blockTag
+  });
+
+  const resp = await fetch(options.apiUrl);
+  const tokenStatus = await resp.json();
+
+  async function checkActivated(address) {
```

Comment:
Any reason to use `async` here? I don't see any async function, i assume you can remove it and also remove the Promise.all on line 55

Reply:
Ah this was leftover from the old code. Removed.